src/main/ts/rules/PreferFun.ts

```diff
@@ -0,0 +1,75 @@
+import { Rule } from 'eslint';
+import { ArrowFunctionExpression, Expression, FunctionExpression, Literal, Statement } from 'estree';
+
+export const preferFun: Rule.RuleModule = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Prefer using the Katamari Fun helper functions, instead of inline functions.'
+    },
+    messages: {
+      preferNoop: 'Prefer using `Fun.noop` instead of `() => {}`',
+      preferAlways: 'Prefer using `Fun.always` instead of `() => true`',
+      preferNever: 'Prefer using `Fun.never` instead of `() => false`'
+    }
+  },
+  create: (context) => {
```

Comment:
Should we add a fixer here? eg to convert `() => {}` to `Fun.noop`

The problem is that we wouldn't be able to add the import if it's not already in the module, which is why I currently haven't added it.

Reply:
I don't think a fixer is important for this